src/main/java/org/spongepowered/common/data/processor/data/tileentity/TileEntityCommandDataProcessor.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.data.processor.data.tileentity;
+
+import com.google.common.collect.Maps;
+import net.minecraft.command.server.CommandBlockLogic;
+import net.minecraft.tileentity.TileEntityCommandBlock;
+import org.spongepowered.api.data.DataContainer;
+import org.spongepowered.api.data.DataHolder;
+import org.spongepowered.api.data.DataTransactionResult;
+import org.spongepowered.api.data.key.Key;
+import org.spongepowered.api.data.key.Keys;
+import org.spongepowered.api.data.manipulator.immutable.ImmutableCommandData;
+import org.spongepowered.api.data.manipulator.mutable.CommandData;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.Texts;
+import org.spongepowered.common.data.manipulator.mutable.SpongeCommandData;
+import org.spongepowered.common.data.processor.common.AbstractTileEntityDataProcessor;
+import org.spongepowered.common.text.SpongeTexts;
+
+import java.util.Map;
+import java.util.Optional;
+
+public class TileEntityCommandDataProcessor extends AbstractTileEntityDataProcessor<TileEntityCommandBlock, CommandData, ImmutableCommandData> {
+
+    public TileEntityCommandDataProcessor() {
+        super(TileEntityCommandBlock.class);
+    }
+
+    @Override
+    public Optional<CommandData> fill(DataContainer container, CommandData commandData) {
+        if (container.contains(Keys.LAST_COMMAND_OUTPUT.getQuery()) || 
```

Comment:
Instead of checking for each individual one, use `container.contains(Keys.FOO.getQuery(), Keys.BAR.getQuery()...)`